altsrc/flag.go

```diff
@@ -109,10 +109,10 @@ func (f *StringSliceFlag) ApplyInputSourceValue(cCtx *cli.Context, isc InputSour
 				continue
 			}
 			underlyingFlag.Value = &sliceValue
-			f.set.Set(n, sliceValue.Serialize())
+			_ = f.set.Set(n, sliceValue.Serialize())
```

Comment:
Lets check the err value explicity and return

Reply:
Issue with checking error return is potential backward compatability issues.

Reply:
If none of the tests fail. I would said lets check it.